mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
FYI there are a few other ways to stop a unit in a tunnel, including but probably not limited to:

* Unloading cargo
* Deploy an MCV/tick tank/etc
* Issuing a move order that fails (e.g. into water)

All of these would go away if the activity cancelling idea works.  If not, we can cover the first two cases by rejecting more orders, and all the other cases by having an idle activity that automatically moves idle actors to the nearest tunnel entrance.